{{cookiecutter.projectDirectory}}/{{cookiecutter.projectName}}/Code/AppDelegate.swift

```diff
@@ -2,14 +2,17 @@ import {{cookiecutter.projectName}}Kit
 import Combine
 import Fetch
 import UIKit
+import Logbook
 
```

Comment:
I would add here a global variable `let log = Logbook.self`
Now you can write `log.info(Environment.current.appInfo)` instead of `Logbook.info(Environment.current.appInfo)`

Reply:
agreed